perl/PullRequestClub.md

```diff
@@ -0,0 +1,32 @@
+Pull Request Club: Stats
+======================================================
+
+Description
+-----------
+
+[Pull Request Club](https://pullrequest.club/) is a web application where users can sign up with their GitHub accounts. Its purpose is to connect
+ - people who are looking for an open source project to work on, with
+ - people who want new contributors for their open source projects.
+
+Pull Request Club gives out monthly assignments. Web application is built on Catalyst (Perl), and it has a SQLite database.
+
```

Comment:
You should maybe move a few "expected outcomes" here as objectives. A project, to be picked up by a student, should have a clear objective or sets of objectives. If you've got a roadmap for the site, point at it; if there are a few milestones, that would help too.